src/client/request.rs

```diff
@@ -46,11 +45,14 @@ impl Request<Fresh> {
     }
 
     /// Create a new client request with a specific underlying NetworkStream.
-    pub fn with_connector<C: NetworkConnector<S>, S: NetworkStream>(method: method::Method, url: Url, connector: &mut C) -> HttpResult<Request<Fresh>> {
-        debug!("{:?} {:?}", method, url);
+    pub fn with_connector<
+        C: NetworkConnector<Stream=S>,
```

Comment:
I don't see how it much cleaner, it looks the same to me except one is before the parens, one is after. But I moved it cause people seem to like it.